www/nginx/Makefile

```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		nginx
-PLUGIN_VERSION=		1.19
+PLUGIN_VERSION=		1.20
```

Comment:
@fabianfrz Is that sufficient for a version bump? I've also listed changes in the changelog.

Reply:
yes